warehouse/query-core/src/test/java/datawave/query/jexl/visitors/UnmarkedBoundedRangeDetectionVisitorTest.java

```diff
@@ -0,0 +1,66 @@
+package datawave.query.jexl.visitors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.apache.commons.jexl3.parser.ASTJexlScript;
+import org.apache.commons.jexl3.parser.ParseException;
+import org.junit.jupiter.api.Test;
+
+import datawave.query.exceptions.DatawaveFatalQueryException;
+import datawave.query.jexl.JexlASTHelper;
+import datawave.query.jexl.visitors.validate.ValidateBoundedRangeVisitor;
+
+public class UnmarkedBoundedRangeDetectionVisitorTest {
+
+    @Test
+    public void testBoundedMarkerWithValidRange() {
+        String query = "((_Bounded_ = true) && (FIELD >= '2' && FIELD <= '3'))";
+        test(false, query);
+    }
+
+    /**
+     * This illegal range is detected by the {@link ValidateBoundedRangeVisitor}
+     */
+    @Test
+    public void testBoundedMarkerWithInvalidRange() {
+        String query = "((_Bounded_ = true) && (FIELD >= '5' && FIELD <= '3'))";
+        test(false, query);
+    }
```

Comment:
shouldn't this also throw an exception?

Reply:
This visitor only checks to see if a bounded range is marked correctly. The BoundedRangeValidationVisitor will handle throwing an exception.